common/src/test/scala/org/neo4j/spark/util/ValidationsIT.scala

```diff
@@ -0,0 +1,180 @@
+package org.neo4j.spark.util
+
+import org.junit.rules.ExpectedException
+import org.junit.{Rule, Test}
+import org.neo4j.driver.AccessMode
+import org.neo4j.spark.{SparkConnectorScalaSuiteIT, TestUtil}
+
+class ValidationsIT extends SparkConnectorScalaSuiteIT {
+
+  val _expectedException: ExpectedException = ExpectedException.none
```

Comment:
why is the extra field needed?

Reply:
It's a workaround otherwise rules does not work. Honestly I don't remember the reason why; please check this discussion (https://stackoverflow.com/questions/32160549/using-junit-rule-with-scalatest-e-g-temporaryfolder)

Reply:
```suggestion
  val expectedException: ExpectedException = ExpectedException.none
```